File: emailApi/node/emailApi.js
var http = require('http');

var email = 'support@whoisxmlapi.com';
var password = 'your_whois_api_password';
var username = 'your_whois_api_username';

var url = 'http://www.whoisxmlapi.com/whoisserver/EmailVerifyService'
        + '?emailAddress=' + email
        + '&validateDNS=true&validateSMTP=true&checkCatchAll=true'
        + '&checkFree=true&checkDisposable=true'
        + '&username=' + username + '&password=' + password

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
}).end();

File: emailApi/python/emailApi.py
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

emailAddress = 'support@whoisxmlapi.com';
password = 'your_whois_api_password'
username = 'your_whois_api_username'

url = 'http://www.whoisxmlapi.com/whoisserver/EmailVerifyService?'\
    + 'emailAddress=' + emailAddress + '&validateDNS=true&validateSMTP=true'\
    + '&checkCatchAll=true&checkFree=true&checkDisposable=true'\
    + '&username=' + username + '&password=' + password

print(urlopen(url).read().decode('utf8'))